app/routes/user/endorsements.server.ts

```diff
@@ -124,6 +128,21 @@ export class EndorsementsServer {
       ConditionExpression:
         'NOT (endorserSub = :endorserSub and requestorSub = :requestorSub and #status <> :status)',
     })
+
+    await sendEmail(
+      'GCN Endorsements',
```

Comment:
GCN Peer Endorsements - I think peer is key so the user understands why they're getting this.